src/tests/unit/doughs-serializer.test.js

```diff
@@ -0,0 +1,120 @@
+import chai from 'chai';
+import config from 'config';
+import _ from 'lodash';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+import { openapi } from 'utils/load-openapi';
+import { doughSerializerData } from './test-data';
+
+const doughAttributes = openapi.definitions.DoughAttributes.properties;
+
+chai.should();
+
+let rawDough;
+let rawDoughs;
+let doughSerializer;
+
+const {
+  nullSpecialInstructionsDough,
+  emptyStringSpecialInstructionsDough,
+} = doughSerializerData;
+
+describe('test doughs serializer', () => {
+  before(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
+    // importing doughSerializer normally causes config to become unextensible,
+    // which then breaks other tests. This prevents that problem, even though
+    // stubbing imports is not required.
+    doughSerializer = proxyquire('../../api/v1/serializers/doughs-serializer', {});
```

Comment:
This was a strange problem to have crop up. I can elaborate if necessary but this seems to have totally fixed it.

Reply:
I ran into the same issue. If I remember right the problem is that importing `doughs-serializer` causes config to run. So you have to replace config before `proxyquire`.

Reply:
Yeah, that was my impression as well. Let me know if there's any issues with the approach I used here.